lighthouse-cli/performance-experiment/server.js

```diff
@@ -94,8 +97,19 @@ function requestHandler(request, response) {
 
 function reportRequestHandler(request, response) {
   try {
+    const id = request.parsedUrl.query.id || defaultReportId;
+
+    const reportsMetadata = Object.keys(database.timeStamps).map(key => {
+      const generatedTime = database.timeStamps[key];
+      return {url: database.url, reportHref: `/?id=${key}`, generatedTime};
```

Comment:
i think can keep the URL building out of here and only in the template for a cleaner view. Here's a proposal (this and the following comments).

```js
return {url: database.url, reportId: key, generatedTime};
```